packages/application-shell/src/components/navbar/tracking-events.ts

```diff
@@ -0,0 +1,17 @@
+export default {
+  collapseMainNav: {
+    event: 'CollapseMainNav',
```

Comment:
Nit: the fields `event` are usually called `action`

Reply:
updated